src/Microplate/Scalars/Row16.php

```diff
@@ -0,0 +1,15 @@
+<?php declare(strict_types=1);
+
+namespace MLL\Utils\Microplate\Scalars;
+
+use MLL\GraphQLScalars\Regex;
+
+class Row16 extends Regex
+{
+    public ?string $description = 'Checks if the given row is of the format A-P-well row';
```

Comment:
Ein Scalar ist ja keine Validierungsfunktion die einen Wert bekommt, sondern repräsentiert den Wert an sich. Vergleiche z.B. https://spec.graphql.org/draft/#sec-Scalars. Zudem würde ich die Bedeutung der 16 erklären.

```suggestion
    public ?string $description = 'Represents a row in a coordinate system with 16 rows. Allowed values range from A-P.';
```